### DISCUSSION TITLE
Inertia packages/repositories naming convention

### DISCUSSION THREAD

**Discussion by Bosphoramus** [inertiajs/inertia#1388] [Ideas]
Now that the front-end adapters have changed the naming convention from, for example

@inertiajs/inertia-vue3

to

@inertiajs/vue3

shouldnt the "inertiajs/inertia-laravel" repo/composer package be named "inertiajs/laravel"?

**Comment by reinink** [comment]
Hah, the thought has certainly crossed my mind.
